### ISSUE TITLE
The device has stopped responding via homebridge

### ISSUE THREAD

**Issue by miguelandres** [lukasroegner/homebridge-dyson-pure-cool#217]
HomeKit says "turning on" but it never does turn on.

I reauthenticated and it didn't work either. There are no weird error logs or anything I can see

```
[11/2/2021, 1:48:33 AM] [DysonPureCoolPlatform] ***-US-******** - set RotationSpeed to 50: {"fnsp":"0005"}
[11/2/2021, 1:48:34 AM] [DysonPureCoolPlatform] ***-US-******** - set RotationSpeed to 100: {"fnsp":"0010"}
[11/2/2021, 1:48:40 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 0: {"fpwr":"OFF","fmod":"OFF"}
[11/2/2021, 1:48:42 AM] [DysonPureCoolPlatform] ***-US-******** - set TargetAirPurifierState to 1 with delay
[11/2/2021, 1:48:42 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 1: setting TargetAirPurifierState cancelled
[11/2/2021, 1:48:42 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 1: {"fpwr":"ON","fmod":"FAN"}
[11/2/2021, 1:48:43 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 0: {"fpwr":"OFF","fmod":"OFF"}
[11/2/2021, 1:48:47 AM] [DysonPureCoolPlatform] ***-US-******** - set TargetAirPurifierState to 1 with delay
[11/2/2021, 1:48:47 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 1: setting TargetAirPurifierState cancelled
[11/2/2021, 1:48:47 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 1: {"fpwr":"ON","fmod":"FAN"}
[11/2/2021, 1:48:54 AM] [DysonPureCoolPlatform] ***-US-******** - set Active to 0: {"fpwr":"OFF","fmod":"OFF"}
```

**Comment by jckstudios** [comment]
Getting the same issue as of two days ago. Did a complete clean install and still not working.

**Comment by ablotsa** [comment]
I am trying to connect my HP02 via Homebridge and getting the same response, constantly says "turning on"

**Comment by lukasroegner** [comment]
As you can already read in resolved issues like #125, #141 and #185, most of the time there is an issue with the network or the IP address (DHCP). Start Homebridge in debug mode (parameter `-D`) and check the logs. If you see recurring "MQTT disconnected/reconnecting" messages, it means that the host of your Homebridge installation has issues communicating with your Dyson device. If that's the case, check the configured IP address of the device. I would always suggest to configure static IP addresses for the Dyson devices.
